eng/pipelines/global-build.yml

```diff
@@ -0,0 +1,51 @@
+trigger: none
```

Comment:
Can we add a comment at the top here to state the purpose of this job? Essentially that it serves to validate that changes don't break the local developer build workflow.

Reply:
Good point, fixed.